src/bump.ts

```diff
@@ -702,20 +710,11 @@ export async function bumpSdkVer(
     `${cv.prefix}${cv.major}.${cv.minor}.${cv.patch}` +
     `${cv.prerelease ? `-${cv.prerelease.replace(/(.+?)\d.*/, "$1")}` : ""}`;
 
-  // See if we already have a dev (draft) release for the _next_ version.
+  // Get the latest draft release matching our current version's prefix.
   // Don't look at the draft version on a release branch; the current version
   // should always reflect the version to be bumped (as no dev releases are
   // allowed on a release branch)
-
-  const latestNextMinorDraft = await getRelease(
-    cv.nextMinor().toString(),
-    true
-  );
-  const latestNextMajorDraft = await getRelease(
-    cv.nextMajor().toString(),
-    true
-  );
-  const latestDraft = latestNextMajorDraft ?? latestNextMinorDraft;
+  const latestDraft = await getRelease(cv.prefix, true);
```

Comment:
After conferring with @KevinDeJong-TomTom, I'll update this to be a regex instead of a "`startsWith()`", which is error-prone.